packages/ui/src/components/editor/EditorBlockBoundary.tsx

```diff
@@ -12,12 +14,18 @@ export const EditorBlockBoundary = memo(function EditorBlockBoundary({ blockEdge
 	const prefix = useClassNamePrefix()
 	return (
 		<div
-			className={cn(`${prefix}editorBlockBoundary`, toEnumViewClass(`${blockEdge}Block`))}
+			className={classNames(
+				`${prefix}editorBlockBoundary`,
+				toEnumViewClass(`${blockEdge}Block`),
+			)}
 			contentEditable={false}
 			data-slate-editor={false}
 			onClick={onClick}
 		>
-			<span className={`${prefix}editorBlockBoundary-inner`}>New paragraph</span>
+			<span className={`${prefix}editorBlockBoundary-inner`}>
+				<Icon blueprintIcon="add" />
+				<Label>New paragraph</Label>
```

Comment:
quick look: this should be translated

Reply:
Agree.

The `I18n` is [coming from the `admin` package](https://github.com/contember/admin/blob/pr/editor-block-boundary-and-handle-alignment-style/packages/admin/src/i18n/useMessageFormatter.ts#L80) and non of the UI components use it directly.

Would providing a prop to pass the formatted message be enough?

Reply:
Yes, totally.